lib/cred_proxy.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python3 -I
+#
+# cred_proxy -- utility functions for checking and using proxies
+#
+# COPYRIGHT 2025 FERMI NATIONAL ACCELERATOR LABORATORY
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+import pathlib
+import shlex
+import subprocess
+import sys
+from typing import Any, Dict, Optional, Union
+
+# TODO: Do we need this anymore since we're IN lib?  # pylint: disable=fixme
+PREFIX = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(os.path.join(PREFIX, "lib"))
```

Comment:
If this file is not being run standalone, it shouldn't need this PREFIX?sys.path.append() code.